package.json

```diff
@@ -52,7 +52,8 @@
     "proxyquire": "0.5.1",
     "grunt-mkdir": "~0.1.1",
     "grunt-contrib-clean": "~0.5.0",
-    "grunt-githooks": "~0.3.1"
+    "grunt-githooks": "~0.3.1",
+    "mongodb": "2.2.20"
```

Comment:
Maybe we should use the same we recently used at https://github.com/telefonicaid/iotagent-node-lib/pull/609/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R50, i.e. 2.2.35

Reply:
Fixed in https://github.com/telefonicaid/sigfox-iotagent/pull/26/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2